util/mesa3d/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017-2020 pal1000
```

Comment:
Who is `pal1000`? The [official license](https://docs.mesa3d.org/license.html) of Mesa states Brian Paul as the copyright holder.

Reply:
He who provides ready-made Windows builds.

Reply:
Does the copyright really get transferred due to building the opensource package?.. I suppose the license on the builder's project page refers to the build scripts, rather than Mesa binaries.

Reply:
I just took the License file as found there. 
Around 2015 I managed to build Mesa myself (involving a cross-compilation on Arch I think). Since 2018 I was unsuccessful, did not need it, and ultimately was happy to find this ready-made release. It's MIT in any case.